src/OpenTelemetry.Api/Internal/BlankSpan.cs

```diff
@@ -39,108 +39,51 @@ private BlankSpan()
         public bool IsRecording => false;
 
         /// <inheritdoc />
-        public Status Status
-        {
-            get => Status.Ok;
-
-            set
-            {
-                if (!value.IsValid)
-                {
-                    throw new ArgumentException(nameof(value));
-                }
-            }
-        }
+        public Status Status { get; set; } = Status.Ok;
 
         /// <inheritdoc />
         public void UpdateName(string name)
         {
```

Comment:
For consistency, we could also log null values for these functions.

Reply:
I've done it on the real span, validation makes little sense on blank one